BotProject/Templates/CSharp/appsettings.json

```diff
@@ -1,4 +1,11 @@
 {
+  "feature": {
+    "UseTelementryLoggerMiddleware": false,
+    "UseTranscriptLoggerMiddleware": false,
```

Comment:
If user set one of these settings to true, how can he know what else he want to configure? I suggest we put the other required settings in the same place with the switch, we can just make them "". Just want to give customer enough clue on this.